code/modules/mob/living/silicon/robot/modules/module_clerical.dm

```diff
@@ -77,7 +77,7 @@
 	E.reagents.add_reagent(/decl/reagent/enzyme, 2 * amount)
 	if(emag)
 		var/obj/item/chems/food/drinks/bottle/small/beer/B = emag
-		B.reagents.add_reagent(/decl/reagent/ethanol/iced_beer, amount)
+		B.reagents.add_reagent(/decl/reagent/ethanol/beer, amount)
```

Comment:
```suggestion
		B.reagents.add_reagent(/decl/reagent/drink/ice, 5)
		B.reagents.add_reagent(/decl/reagent/ethanol/beer, 20)
```

Reply:
This one should still be multiples of amount, probably `amount` and `amount/2` respectively so the total mix is 20% ice